### ISSUE TITLE
PHPCS parser wrong class name

### ISSUE THREAD

**Issue by true-hero** [DefectDojo/django-DefectDojo#4216]
### Bug description
When you try to import phpcs report via api, you have 500 response code with error in logs:
```
uwsgi_1         | None
uwsgi_1         | ERROR:django.request:Internal Server Error: /api/v2/import-scan/
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "./dojo/api_v2/serializers.py", line 1097, in save
uwsgi_1         |     parser_findings = parser.get_findings(data.get('file', None), test)
uwsgi_1         |   File "./dojo/tools/php_security_audit_v2/parser.py", line 39, in get_findings
uwsgi_1         |     sev = PhpSecurityAuditV2.get_severity_word(issue["severity"])
uwsgi_1         | NameError: name 'PhpSecurityAuditV2' is not defined
```

### Steps to reproduce
1. Get phpcs report file with findings
2. Try to upload it via API

### Possible fix
I guess, the problem is in class name [here](https://github.com/DefectDojo/django-DefectDojo/blob/release/1.14.0/dojo/tools/php_security_audit_v2/parser.py#L39). It should be called like:
```php
sev = PhpSecurityAuditV2Parser.get_severity_word(issue["severity"])
```

### Environment information
 - DefectDojo 1.14.0

### Expected behavior
PHPCS report file should be imported without errors.

**Comment by whitespots** [comment]
@damiencarol Hello! I suppose this issue maybe somehow related to https://github.com/DefectDojo/django-DefectDojo/issues/4158

**Comment by damiencarol** [comment]
@whitespots taking a look

**Comment by damiencarol** [comment]
Fixed in `dev` / `1.15.0`
